src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1441,7 +1510,32 @@ ExpressionPtr Visitor::visitTypesafe(
 }
 
 // ____________________________________________________________________________________
-ExpressionPtr Visitor::visitTypesafe(Parser::AggregateContext* ctx) {
+void Visitor::visit(Parser::RegexExpressionContext* ctx) {
+  reportError(ctx, "The REGEX built in function is not yet implemented.");
+}
```

Comment:
I think in these error it suffices to say `The REGEX function...` (without the `built in` which is merely a technical detail of no interest to the user).